packages/entities/entities-vaults/src/components/VaultList.vue

```diff
@@ -402,6 +402,19 @@ const hideDeleteModal = (): void => {
   isDeleteModalVisible.value = false
 }
 
+const deleteAssociatedConfigStore = async (configStoreId: string): Promise<void> => {
+  const { apiBaseUrl, app, controlPlaneId } = (props.config as KonnectVaultListConfig)
+  const url = `${apiBaseUrl}${endpoints.list[app].deleteConfigStore}`
+    .replace(/{controlPlaneId}/gi, controlPlaneId || '')
+    .replace(/{id}/gi, configStoreId)
+  try {
+    await axiosInstance.delete(url)
+  } catch (error: any) {
+    // There is a rare case where more than 1 vaults are linked to the config store (i.e. created via API).
+    // In this case, the deletion will fail but the UI can safely ignore the error.
+  }
+}
```

Comment:
qq: Would we delete the associated config store if a user tries to delete a Konnect vault from the vault detail page as well?

Reply:
Yes I think we should. I'll add this logic in the host app.